editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -170,8 +169,8 @@ export interface EditorStore extends Store.Store {
   ) => Promise<string>;
   createFlowFromTemplate: (
     templateId: string,
-    teamId: string,
-  ) => Promise<object>;
+    teamId: number,
+  ) => Promise<AxiosResponse>;
```

Comment:
```suggestion
  ) => Promise<AxiosResponse<{ id: string; slug: string; message: string }>>;
```

nit: We can type this to match `CreateFlowFromTemplateResponse`